fe1-web/components/UserListItem.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  View, Text, StyleSheet, ViewStyle, TextStyle,
+} from 'react-native';
+import { getUserSocialChannel, Hash, PublicKey } from 'model/objects';
+import { gray } from 'styles/colors';
+import { Ionicons } from '@expo/vector-icons';
+import STRINGS from 'res/strings';
+import { subscribeToChannel } from 'network/CommunicationApi';
+import WideButtonView from './WideButtonView';
+
+/**
+ * Component that shows a user's profile picture, his public key and two buttons:
+ * - Follow, which subscribes to the user's social channel.
+ * - Profile, that navigates to the user's profile page.
+ *
+ * @remarks
+ * For now, it is not possible to unfollow users, and you have to follow someone to access to
+ * his profile.
```

Comment:
```suggestion
 * For now, it is not possible to unfollow users, and you have to follow someone to access
 * their profile.
```
Inclusive 😜 

Also, in general, I would always recommend indicating a date (and possibly a username) whenever there's a comment with a notion of time.
This makes it easy to revisit / challenge the assumption if a lot of time has passed.
E.g., if you see code that says "We probably need to fix this soon", you won't react and interpret it in the same way if the full comment says "... fix this soon (2016-01-01, xyz)" or "... fix this soon (2021-11-19, jiabaow)".

Reply:
Okay, I'll do it ! It will make things clearer